app/services/migration/migrators/user.rb

```diff
@@ -44,8 +44,16 @@ def call
 
   private
 
-    def validate_multiple_trns!(unique_trns, user)
-      return unless unique_trns.size > 1
+    def unique_validated_trns(ecf_user)
+      ecf_user.npq_applications
```

Comment:
could we include npq applications in the query, so there are no n+1s?

Reply:
I don't think that would make any difference here as - as confusing and unintuitive as it is - to be consistent with ECF behaviour `npq_applications` runs a separate query:

https://github.com/DFE-Digital/npq-registration/blob/main/app/models/migration/ecf/user.rb#L8-L10

Reply:
ah gotcha 👍